### PR TITLE
Feature/settings string review

### PR DESCRIPTION
**Pull request by njjc**

### CODE REVIEW

**Comment by davidhabib** [comment]
@dospromptman @njjc this branch is failing with references in the translation file to fields that don't exist.

**Comment by njjc** [comment]
@davidhabib this is ready for review. The build marked failed is a false negative, if you click through the status is good.

**Comment by njjc** [comment]
I'll do the update to api 36 in a different branch.

File: src/package.xml [hunk 1]
```diff
@@ -177,11 +178,6 @@
         <name>RemoteSiteSetting</name>
     </types>
     <types>
-        <members>Nonprofit_Edition_Reports</members>
```
Comment (davidhabib) [inline src/package.xml]: did you mean to remove these reports from the package?  are we sure that is safe?

**Comment by davidhabib** [comment]
@njjc why are we removing the reports?  thanks.

**Comment by njjc** [comment]
@davidhabib they're referencing the fields we're deprecating, and will be removing next time we rev the underlying packages, so they can't exist past that point. Removing them from the package doesn't affect orgs that currently have them, they just won't be included in new installs.

**Comment by davidhabib** [comment]
@njjc so sounds like people are ok with us removing these reports, but you mentioned that one of the other packages uses this report folder?  does that mean you still need to modify this pull request?

**Comment by njjc** [comment]
@davidhabib this is good to go, we should verify that the reports folder remains in the packaging org once this is merged in and the build process finished. It worked correctly in my local dev org, but I want us to check to be safe.

**Pull request merged**